generate_ics.py

```diff
@@ -62,11 +69,16 @@ def create_ics(input_filename = None):
 
 
 if __name__ == "__main__":
-    if(len(sys.argv) == 1):
-        create_ics()
-    elif(len(sys.argv) > 1):
-        if(path.exists(sys.argv[1]) and path.isfile(sys.argv[1])):
-            create_ics(sys.argv[1])
+    if(args.input_file is not None and path.exists(args.input_file) and path.isfile(args.input_file)):
+        if(args.output_file is not None):
+            create_ics(output_filename=args.output_file, input_filename=args.input_file)
         else:
-            print(f"File {sys.argv[1]} does not exist. Going into console input mode.")
-            create_ics()
+            create_ics(args.input_file)
```

Comment:
I don't think this case will ever be hit because `--output-file` has a default (so it's not much of a problem), but I think the current line would output to the input file and kick into console mode which is not what we want

Suggest getting the default explicitly to use as output_filename

Reply:
You are right about this. I updated the lines, so that the output-file argument doesn't get checked. Now it just checks if the input file was given or not. I also made two different error messages. One if the user didn't specify the input file at all, and one if the user specified an input file, which couldn't be found by the script.